internal/handler/handler.go

```diff
@@ -33,6 +33,81 @@ type userMgmtSvc struct {
 	logic logic.UserMgmtSvcLogicIer
 }
 
+func validatePass(pass string) *respModel.Response {
+	if len(pass) < 8 {
+		return &respModel.Response{
+			Status:  http.StatusBadRequest,
+			Message: "Password must be 8 characters long",
+			Data:    nil,
+		}
+
+	}
+	done, err := regexp.MatchString("([a-z])+", pass)
+	if err != nil {
+		log.Error(err)
+		return &respModel.Response{
+			Status:  http.StatusInternalServerError,
+			Message: "Failed to match password",
+			Data:    nil,
+		}
+	}
+	if !done {
+		return &respModel.Response{
+			Status:  http.StatusBadRequest,
+			Message: "Password must contain 1 lower case character",
+			Data:    nil,
+		}
+	}
+	done, err = regexp.MatchString("([A-Z])+", pass)
+	if err != nil {
+		log.Error(err)
+		return &respModel.Response{
+			Status:  http.StatusInternalServerError,
+			Message: "Failed to match password",
+			Data:    nil,
+		}
+	}
+	if !done {
+		return &respModel.Response{
+			Status:  http.StatusBadRequest,
+			Message: "Password must contain 1 upper case character",
+			Data:    nil,
+		}
+	}
+	done, err = regexp.MatchString("([0-9])+", pass)
+	if err != nil {
+		log.Error(err)
+		return &respModel.Response{
+			Status:  http.StatusInternalServerError,
+			Message: "Failed to match password",
+			Data:    nil,
+		}
+	}
+	if !done {
+		return &respModel.Response{
+			Status:  http.StatusBadRequest,
+			Message: "Password must contain 1 numeric character",
+			Data:    nil,
+		}
+	}
+	done, err = regexp.MatchString("([@_])+", pass)
```

Comment:
Other special characters also permitted. Please check the requirement and update.

Reply:
done